### PR TITLE
Integration of api into app

### PR DESCRIPTION
**Pull request by yhbrandon**
- Integrating iceman into the app
- Update authorization to check for token and bypassing login
- Routing

### CODE REVIEW

File: src/core/utils/checkAuth.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const isAuthorized = () => {
+  return !!sessionStorage.jwt
+}
+
+export default isAuthorized
```
Comment (fmaredia) [inline src/core/utils/checkAuth.js]: newline needed

File: src/core/utils/fetchJSON.js [hunk 2]
```diff
@@ -24,9 +25,11 @@ const fetchJSON = (endpoint, method, body) => {
     headers: {
       'Accept': 'application/json',
       'Content-Type': 'application/json'
-    }
+    },
```
Comment (fmaredia) [inline src/core/utils/fetchJSON.js]: why the comma?

File: src/modules/Auth/Auth.js [hunk 3]
```diff
@@ -9,41 +9,27 @@ import HeaderLogo from 'assets/oh-hi_Logo_2-03.svg'
...
+Auth.propTypes = {
+  email: PropTypes.string,
+  handleChange: PropTypes.func,
+  handleSubmit: PropTypes.func
 }
 
-
-export default AuthForm
\ No newline at end of file
+export default Auth
```
Comment (fmaredia) [inline src/modules/Auth/Auth.js]: newline needed

File: src/modules/Auth/AuthContainer.js [hunk 4]
```diff
@@ -9,8 +9,56 @@ import Auth from './Auth'
...
+  }
+
+  render () {
+    const { email, type } = this.props
+    let showVerification = email && type === 'submit'
+
+    return (
+      <div>
+        { showVerification && <Verification email={ email } /> }
+        { !email &&
```
Comment (fmaredia) [inline src/modules/Auth/AuthContainer.js]: i haven't seen this syntax before, p cool

File: src/core/utils/fetchJSON.js (hunk 2 above)
Comment (oh-hi) [inline src/core/utils/fetchJSON.js]: Leftover from a removed prop. Will remove

File: src/modules/ProfileEditor/ProfileEditorContainer.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+const mapPropsToState = ({ api: { user: { data = {} } } }) => {
+  return { data }
+}
+
+const mapDispatchToProps = (dispatch) => {
+  return bindActionCreators(Actions, dispatch)
+}
+
+export default connect(mapPropsToState, mapDispatchToProps)(ProfileEditorContainer)
```
Comment (fmaredia) [inline src/modules/ProfileEditor/ProfileEditorContainer.js]: newline needed

File: src/views/Signup/Signup.js [hunk 6]
```diff
@@ -1,11 +1,11 @@
...
+// Modules
 import Auth from 'modules/Auth/AuthContainer'
-import Verification from 'modules/Verification/Verification'
 
 const Signup = ({ email }) => {
-  return email ? (<Verification email={ email } />) : (<Auth type='register' />)
+  return <Auth type='register' />
 }
 
 export default Signup
```
Comment (fmaredia) [inline src/views/Signup/Signup.js]: newline needed

**Comment by oh-hi** [comment]
The story of this is, I need to get linting working

**Pull request merged**